compiler/cbmc/src/lib.rs

```diff
@@ -32,3 +32,7 @@ mod machine_model;
 pub mod utils;
 pub use irep::serialize;
 pub use machine_model::{MachineModel, RoundingMode};
+mod cbmc_string;
+pub use cbmc_string::{InternString, InternStringOption, InternedString};
+
+pub const NO_PRETTY_NAME: Option<InternedString> = None;
```

Comment:
Is it Rust-ey to use this over just None?

Reply:
I believe you have to be explicit about the type here or rustc won't be able to infer the type.

Reply:
Celina is correct. Rust can't infer the type, so either you use this, or the turbofish.  I'll add a comment.